src/uct/api/uct_def.h

```diff
@@ -137,6 +144,25 @@ typedef struct uct_iov {
 } uct_iov_t;
 
 
+/**
+ * @ingroup UCT_RESOURCE
+ * @brief Data received from the remote peer.
+ *
+ * The remote peer's device address, the data received from it and their lengths.
+ * Used with the client-server API on a connection manager.
+ */
+typedef struct uct_cm_remote_data {
+    const uct_device_addr_t *remote_dev_addr;       /**< Address of the remote peer */
```

Comment:
Since the struct name has "remote", can be just `dev_addr` and `dev_addr_length`, also need to point in the comment that this is "device address".